.github/workflows/deploy-prod.yml

```
@@ -83,7 +83,23 @@
+          elif [ -f "working-kubeconfig.yaml" ]; then
+            cp working-kubeconfig.yaml kubeconfig
+            echo "Using working-kubeconfig.yaml from repository"
+          else
+            echo "No valid kubeconfig found in repository"
+            exit 1
+          fi
+        else
+          echo "Using KUBE_CONFIG secret"
+          echo "${{ secrets.KUBE_CONFIG }}" | base64 -d > kubeconfig
+        fi
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)